notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Could you add hyperlinks to the keras document for each keras_nlp module in markdown? </p><p>https://keras.io/api/keras_nlp/tokenizers/word_piece_tokenizer/</p><p> </p><p>Same for other modules.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done!</p>